.gitignore

```diff
@@ -18,9 +18,6 @@ dist/
 coverage/
 node_modules/
 
-# worker in backend resources
```

Comment:
why this is removed?

Reply:
src/backend/backend-loader/src/main/resources/activepieces-worker.js this is from old backend